content/src/collections/f82c2201-b35a-47f0-a2ad-26895e3fa0b6.json

```diff
@@ -0,0 +1,38 @@
+{
+  "en": {
+    "id": "f82c2201-b35a-47f0-a2ad-26895e3fa0b6",
+    "name": "Test Playlist",
+    "exercises": [
+      "095f9642-73b6-4c9a-ae9a-ea7dea7363f5",
+      "5b0dea9f-c665-41ce-a779-601d0cfa433d",
+      "3188a5ed-a1d6-451c-ae2a-f44f4df37495"
+    ],
+    "image": {
+      "source": "https://res.cloudinary.com/cupcake-29k/image/upload/q_auto,t_global/v1676639667/Images/placeholder_image_card_nxt9yf.png"
+    }
+  },
+  "pt": {
+    "id": "f82c2201-b35a-47f0-a2ad-26895e3fa0b6",
+    "exercises": [
+      "095f9642-73b6-4c9a-ae9a-ea7dea7363f5",
+      "5b0dea9f-c665-41ce-a779-601d0cfa433d",
+      "3188a5ed-a1d6-451c-ae2a-f44f4df37495"
+    ]
+  },
+  "sv": {
+    "id": "f82c2201-b35a-47f0-a2ad-26895e3fa0b6",
+    "exercises": [
+      "095f9642-73b6-4c9a-ae9a-ea7dea7363f5",
+      "5b0dea9f-c665-41ce-a779-601d0cfa433d",
+      "3188a5ed-a1d6-451c-ae2a-f44f4df37495"
+    ]
+  },
+  "es": {
+    "id": "f82c2201-b35a-47f0-a2ad-26895e3fa0b6",
+    "exercises": [
+      "095f9642-73b6-4c9a-ae9a-ea7dea7363f5",
+      "5b0dea9f-c665-41ce-a779-601d0cfa433d",
+      "3188a5ed-a1d6-451c-ae2a-f44f4df37495"
+    ]
+  }
+}
```

Comment:
Should we maybe remove these test collections for now?

Reply:
Yes, they was just for me testing, will remove them. Totally forgot that too 😂